webapp/app/partials/perf/comparectrl.html

```diff
@@ -0,0 +1,42 @@
+<nav class="navbar navbar-default" role="navigation">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="#">Perfherder Compare Talos</a>
+    </div>
+  </div>
+</nav>
+
+<div class="ph-horizontal-layout">
+  <div id="revision-chooser">
+   &nbsp;
+  </div>
+  <div id="data-display">
+    <div id="datapoint-detail" ng-show="originalRevision && newRevision">
+      <div ng-if="dataLoading">
+        Loading all results, please wait a minute...
+        <img src="img/dancing_cat.gif" />
+      </div>
+      <div id="subtest-summary" ng-if="!dataLoading">
+        <h4>Compare All Data for Revision {{newProject}} : {{newRevision}} to {{originalProject}} : {{originalRevision}}</h4>
```

Comment:
Some spacing and capitalization issues here. I'd prefer:

Compare all data for revision {{newProject}}: to {{originalProject}}: {{originalRevision}}